main.py

```diff
@@ -30,10 +30,6 @@ def load() -> None:
         bot.load_extension(f"cogs.{cog}")
 
 
-def main() -> None:
+if __name__ == "__main__":
     load()
     bot.run(TOKEN)
-
-
-if __name__ == "__main__":
-    asyncio.run(main())
```

Comment:
you can't `asyncio.run(None)` (which is what `main()` returns)
Not totally sure what the motivation was for using asyncio.run in the first place, `bot.run()` handles all of the asyncio stuff internally.